dockstore-common/pom.xml

```diff
@@ -298,8 +298,8 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.scala-tools</groupId>
-                <artifactId>maven-scala-plugin</artifactId>
+                <groupId>net.alchim31.maven</groupId>
```

Comment:
Got a blog post or issue talking about the difference between these two?

Reply:
https://docs.scala-lang.org/tutorials/scala-with-maven.html "We’ll be using the Scala Maven Plugin (GitHub repo, website) (formerly known as the maven-scala-plugin; renamed to honour the new naming policy where only Maven core plugins are prefixed with “maven”),"